### PR TITLE
fix: display initial for tokens without image

### LINKED ISSUE #3769: Improve manual token addition
Reported by Olga

The icon looks broken (or looks like is forbidden token)

Image

Ideally we improve also the image finding, and how we help the user assess if its the right token. Coingecko link or similar?

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Fixes #3769 

Show token initial when there's no image url

|light|dark|
|-|-|
|Screenshot 2024-04-11 at 17 29 38 |Screenshot 2024-04-11 at 17 29 27|

# To Test

1. Open token selector
2. Scroll or search for a token that has no image
* Initial should be displayed instead of the ⊘ icon

### CODE REVIEW

File: libs/tokens/src/pure/TokenLogo/SingleLetterLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { UI } from "@cowprotocol/ui"
+import styled from "styled-components/macro"
```
Comment (shoom3301) [inline libs/tokens/src/pure/TokenLogo/SingleLetterLogo.tsx:2]: Not a big deal, but double quotes here :)

**Review by shoom3301 — APPROVED** [review]
Simple and neat!

File: libs/tokens/src/pure/TokenLogo/SingleLetterLogo.tsx (hunk 1 above)
Comment (alfetopito) [reply libs/tokens/src/pure/TokenLogo/SingleLetterLogo.tsx:2]: Damn vscode.
Still not working quite right yet.
Will fix it.

**Review by elena-zh — COMMENTED** [review]
Hey @alfetopito , great!!
However, some nitpicks with buttons alignment inside circles. See images: 
shifted 2
shifted
Also, it would be great not to underline a letter when hover a mouse on a token on the Account-Tokens page :) 
underline

**Comment by alfetopito** [comment]
@elena-zh other than the iOS issues, I could reproduce the issues.
Let's hope the fix for them also fixes it on iPhone 🤞

**Comment by elena-zh** [comment]
Hey @alfetopito , issues with a letter alignment are fixed, thank you!
One more tiny issue is not addressed: the letter is underlined on hover on Tokens page. Could you please remove an underlining from the icon?
underline1

**Pull request merged**